presto-verifier/src/main/java/com/facebook/presto/verifier/framework/QueryConfigurationOverridesConfig.java

```diff
@@ -107,6 +110,28 @@ public QueryConfigurationOverridesConfig setSessionPropertiesOverrideStrategy(Se
         return this;
     }
 
+    @Override
+    public Set<String> getSessionPropertiesRemoval()
+    {
+        return sessionPropertiesRemoval;
+    }
+
+    @Config("session-properties-removal")
+    public QueryConfigurationOverridesConfig setSessionPropertiesRemoval(String sessionPropertyRemoval)
+    {
+        if (sessionPropertyRemoval == null) {
+            return this;
+        }
+
+        try {
+            this.sessionPropertiesRemoval = new ObjectMapper().readValue(sessionPropertyRemoval, new TypeReference<Set<String>>() {});
+        }
+        catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
should we just say something like `;` separated values ? That way specifying it is easier and the logic here is simpler as well

Reply:
I prefer comma separated as well, and  I can use Splitter and do that. 
But for Map above we are using Json Serialized format. Is that a standard for PrestoDB?

Reply:
For JDBC URI parameters we separate key and value with `=` and different key-value pairs are separated by `;`.

Reply:
✅